docs/docs/application-basics.md

```diff
@@ -107,9 +108,9 @@ A typical pattern in the development and deployment cycle is to have your automa
 
 ## A Simple Docker-based Application
 
-With Marathon it is straightforward to run applications that use Docker images. See also [Running Docker Containers on Marathon]({{ site.baseurl }}/docs/native-docker.html) for further details and advanced options.
+It is straightforward to run applications that use Docker images in Marathon. See [Running Docker Containers on Marathon]({{ site.baseurl }}/docs/native-docker.html) for further details and advanced options.
 
-In the following example application definition, we will focus on a simple Docker app: a Python-based web server using the image [python:3](https://registry.hub.docker.com/_/python/). Inside the container, the web server runs on port `8080` (the value of `containerPort`). Outside of the container, Marathon assigns a random port (`hostPort` is set to `0`):
+In the following example application definition, we will focus on a simple Docker app: a Python-based web server using the image [python:3](https://registry.hub.docker.com/_/python/). Inside the container, the web server runs on port `8080` (the value of `containerPort`). Outside of the container, Marathon assigns a random port (`hostPort` is set to `0`): <!-- does this mean that if you set hostPort to 0 you'll get assigned a random port? -->
```

Comment:
Does the last sentence mean that if you set hostPort to 0 you'll get assigned a random port?